app/elements/candidates/invite-candidate-input.html

```diff
@@ -71,13 +83,31 @@
 				isSingleChallenge: {
 					type: Boolean,
 					computed: '_isSingleChallenge(challenges)'
+				},
+				selectClass: {
+					type: String,
+					value: 'self-center'
+				}
+			},
+			selectClass: function(){
+				return this.invalidSelect ? 'invalid self-center' : 'self-center';
+			},
+			validate: function(){
+				var valid = true;
+				//TODO: When https://github.com/PolymerElements/paper-dropdown-menu/pull/57 gets released, revisit this.
```

Comment:
I just left them a comment. Releasing a new version should not be too cumbersome, maybe they are nice enough to release this in the next days.